demo-app/src/androidTest/java/com/worldpay/access/checkout/sample/card/standard/testutil/CardFragmentTestUtils.kt

```diff
@@ -210,4 +210,11 @@ class CardFragmentTestUtils(activityRule: ActivityTestRule<MainActivity>) : Abst
         wait(maxWaitTimeInMillis = 20000) { assertEquals(cardBrand.cardBrandName, brandLogo().getTag(R.integer.card_tag)) }
         return this
     }
+
+    fun autofillHints(input: String): CardFragmentTestUtils {
```

Comment:
Can we keep the  naming consistent? `autofillHints` => `hasAutofillHints`

Reply:
We pass `input` as an argument but never use it whereas we should according to the code in `shouldHaveCorrectAutofillHints` in `CardFragmentTest`